packages/core/src/external/cda/process-attachments.ts

```diff
@@ -126,7 +156,7 @@ async function handleFhirUpload(
   fhirUrl: string,
   log: typeof console.log
 ): Promise<void> {
-  log(`Transaction bundle: ${JSON.stringify(transactionBundle)}`);
+  log(`Transaction bundle would be: ${JSON.stringify(transactionBundle.entry)}`);
```

Comment:
Not I agree with the `would be` -- it makes it sound like this is a dry run (which it's not).

Reply:
lol woops i made this change just for local testing and i forgot to revert it..